perses/app/relative_setup.py

```diff
@@ -193,23 +230,20 @@ def __init__(self,
                 self._nonbonded_method = app.NoCutoff
 
 
-        self._trajectory_prefix = trajectory_prefix
-        self._trajectory_directory = trajectory_directory
 
         beta = 1.0 / (kB * temperature)
 
-        mol_list = []
+        self.mol_list = []
```

Comment:
I wonder if we really want this to be a public attribute. Do we want to allow users to change this? I think for the `_setup_complex` method this doesn't need to be public.

Reply:
That changes has nothing to do with making it public or private but instead fixes this subtle bug:
```
>>> class Foo():
...  my_list = []
... 
>>> foo = Foo()
>>> foo.my_list.append(1)
>>> bar = Foo()
>>> bar.my_list
[1]
>>>
```

Reply:
I think that applies only for class attributes, but as far as I can see this is just a variable in the `__init__` function. I am okay either way, but just being mindful of not having too many public attributes.